src/plone/restapi/serializer/dxcontent.py

```diff
@@ -39,6 +40,18 @@
     WorkingCopyInfo = None
 
 
+def get_allow_discussion_value(context, request, result):
+    # This test is to handle the plone.app.discussion not being installed situation
+    if "allow_discussion" in result:
+        # Check if the content item implements the IContentish interface
+        if IContentish.providedBy(context):
+            result["allow_discussion"] = getMultiAdapter(
+                (context, request), name="conversation_view"
+            ).enabled()
+    else:
+        return False
```

Comment:
@Akshat2Jain the else that must have `result["allow_discussion"] = False` is this.